nitrates/analysis_seeds/UpperLimits.py

```diff
@@ -0,0 +1,362 @@
+   
+        max_sig2_bkg = np.max(sig2_bkg_values)
+        #print('max of sig2_bkg=',max_sig2_bkg)
+        #print('sig2_bkg=',sig2_bkg)
+        logging.debug('max sig2_bkg:')
+        logging.debug(max_sig2_bkg) 
+        logging.debug('bkg rate:')
+        logging.debug(bkg_rate)
+        logging.debug('bkg rate error:')
+        logging.debug(bkg_rate_err)
+
+
+# UL
+
+
+        Ndets_tot = 32768.0 # total dets 
+        Ndets_active = ndets   # number of active detectors
+        #print('No. of avtive detectors =',ndets)
+
+        Ndet_ratio = Ndets_active / Ndets_tot
+    
+# new using new responses ----
+
+        theta_values=["0.000","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","109.471","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","120.000","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","131.810","144.341","144.341","144.341","144.341","144.341","144.341","144.341","144.341","144.341","144.341","144.341","144.341","156.444","156.444","156.444","156.444","156.444","156.444","156.444","156.444","16.699","16.699","168.284","168.284","168.284","168.284","180.000","21.801","21.801","26.565","26.565","26.565","26.565","30.964","30.964","35.796","35.796","35.796","35.796","38.660","38.660","40.511","40.511","40.511","40.511","41.987","41.987","44.564","44.564","44.564","44.564","45.000","45.000","45.000","45.000","50.194","50.194","50.194","50.194","50.292","50.292","50.292","50.292","51.046","51.046","51.046","51.046","51.671","51.671","51.671","51.671","53.301","53.301","53.301","53.301","55.264","55.264","55.264","55.264","56.310","56.310","56.310","56.310","56.310","56.310","57.212","57.212","57.212","57.212","57.995","57.995","58.438","58.438","58.438","58.438","59.491","59.491","59.491","59.491","59.534","59.534","59.534","59.534","59.664","59.664","59.664","59.664","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","60.000","61.421","61.421","61.421","61.421","62.499","62.499","62.499","62.499","63.435","63.435","63.435","63.435","65.489","65.489","65.489","65.489","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","70.529","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","80.406","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","90.000","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594","99.594"]    
```

Comment:
store these theta_phi values pairwise in a file

Reply:
oh you already have them in a file? why are they here then?

Reply:
the theta phis are now being directly read out from the resp_dir fnames, as Jimmy showed.